src/w2form.js

```diff
@@ -789,6 +789,28 @@
             setTimeout(function () { w2utils.unlock(box, speed); }, 25); // needed timer so if server fast, it will not flash
         },
 
+        lockPage: function(page, msg){
```

Comment:
I usually add in comments on top of the file what was added. It is needed to update documentation later.